docs/manual/system/settings.en.md

```diff
@@ -544,12 +544,12 @@ of the `config/parameters.yaml` and thus you should not use these parameters, if
 
 The following example shows how to define the system administrator's e-mail address using a custom environment variable in the `.env` file and referencing it in the `config.yml` file.
 
-`yaml
+```dotenv
```

Comment:
```suggestion
```yaml
```
The usage of `yaml` was deliberate, to have some syntax highlighting (so that comments are coloured differently from the rest).

Reply:
The `.env`  file is not a YAML file and `dotenv` type should highlight comments as well.

As there are no comments in this example. the markdown type shouldn't matter that much.

Reply:
As far as I checked, there is no `dotenv` type. No syntax highlighting will take place if you use `dotenv`.

Reply:
```suggestion
```ini
```

@fritzmg I would suggest to use `ini` instead. This is supported and has more or less the same valid syntax.
![grafik](https://user-images.githubusercontent.com/544986/211944192-9cc714c3-5767-4cb9-8845-d43eb9d138cb.png)

Reply:
Yes, that's fine with me :). Please also adjust the German version of the page accordingly.

Reply:
@fritzmg Thank you very much for your feedback and review of the changes.